persistence/src/main/scala/za/co/absa/spline/persistence/ArangoInit.scala

```diff
@@ -140,4 +145,35 @@ object ArangoInit extends ArangoInit {
         val functionBody = ARM.using(Source.fromURL(res.getURL))(_.getLines.mkString)
         db.createAqlFunction(functionName, functionBody, new AqlFunctionCreateOptions()).toScala
       }))
+
+  private val searchViewName = "attributeSearchView"
+
+  private def deleteViews(db: ArangoDatabaseAsync) = {
+    val view = db.view(searchViewName)
```

Comment:
This line throws an exception if the view doesn't exist.

Reply:
Why do you think so? It does not throw any exceptions for me. It would not make sense to have exists method on the view if it did.

Reply:
Hm, you are right. It was throwing for me, but I cannot reproduce it anymore :(
Please ignore this one then.